frontend/src/RecordingPageComp/SimilarSounds/SimilarSounds.jsx

```diff
@@ -1,28 +1,38 @@
 /* eslint-disable react/prop-types */
 
-import { useState } from "react";
+import { useState, useEffect } from "react";
 import "./SimilarSounds.css";
 
-function SimilarSounds({ playbackDuration }) {
+function SimilarSounds({ note, playbackDuration }) {
   const duration = Math.floor(playbackDuration);
   const [soundList, setSoundList] = useState([]);
   const [isLoadingSounds, setIsLoadingSounds] = useState(false);
 
-  const fetchSounds = async () => {
+  useEffect(() => {
+    const midiNote = Math.floor(note);
```

Comment:
Does the note need to be an integer for the API request?

Reply:
Yup. The note variable is a float because it is the result of an average. MIDI notes are supposed to be integer.